### PR TITLE
Update documentation for video tracks

### PR DESCRIPTION
**Pull request by kubaau**

### CODE REVIEW

File: docs/static/img/on_video_track_added.png
Comment (Adam1-3) [inline docs/static/img/on_video_track_added.png:1]: we could improve look&feel of some diagrams like this one

File: docs/static/img/get_texture.PNG
Comment (Adam1-3) [inline docs/static/img/get_texture.PNG:1]: just using Reroute Nodes and a bit of align would improve look&feel :)

**Review by Adam1-3 — APPROVED** [review]
if you don't want to polish diagrams now please add a task for it so we did it later.

**Comment by kubaau** [comment]
> if you don't want to polish diagrams now please add a task for it so we did it later.

I added one to the project.

**Pull request merged**